source/stylist/fortran.py

```diff
@@ -11,7 +11,8 @@
 from collections import defaultdict
 from typing import Dict, List, Optional, Pattern, Type, Union
 
-import fparser.two.Fortran2003 as Fortran2003  # type: ignore
+import fparser.two.Fortran2003 as Fortran2003
```

Comment:
fparser does not implement type hinting so for the moment must be marked as ignored. This is why your change is failing the "mypy" tests.

Reply:
I've put this back in, not sure why I deleted it in the first place